Engines/Wine/Verbs/dotnet20sp2/script.js

```diff
@@ -0,0 +1,76 @@
+include("engines.wine.engine.object");
+include("engines.wine.plugins.override_dll");
+include("utils.functions.net.resource");
+include("utils.functions.filesystem.files");
+include("engines.wine.plugins.windows_version");
+include("engines.wine.verbs.remove_mono");
+
+
+/**
+* Verb to install .NET 2.0 SP2
+* @returns {Wine} Wine object
+*/
+Wine.prototype.dotnet20sp2 = function () {
+    var osVersion = this.windowsVersion();
+    this.windowsVersion("winxp");
+    var dlls = [
+        "ngen.exe",
+        "regsvcs.exe",
+        "mscorsvw.exe"];
+    this.overrideDLL()
+        .set("builtin", dlls)
+        .do();
+
+    if (this.architecture() == "x86") {
+
+        var setupFile32 = new Resource()
+            .wizard(this.wizard())
+            .url("https://download.microsoft.com/download/c/6/e/c6e88215-0178-4c6c-b5f3-158ff77b1f38/NetFx20SP2_x86.exe")
+            .checksum("22d776d4d204863105a5db99e8b8888be23c61a7")
+            .name("NetFx20SP2_x86.exe")
+            .get();
+
+        this.removeMono();
```

Comment:
You should remove mono at the beginning, not in each if elif else.

Reply:
`dotnet20` have similiar structure (calling `removeMono` in every if/else)

Reply:
Well, it is BAD ^^.

Reply:
How so?